ui/app/routes/vault/cluster/secrets/backend/credentials.js

```diff
@@ -16,10 +16,6 @@ export default Route.extend({
   router: service(),
   store: service(),
 
-  backendModel() {
```

Comment:
consolidated this logic below, I didn't see this route extended anywhere or this method uses again in this route. but i can revert!